### PR TITLE
Test travis for php7

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

**Comment by rdoursenaud** [comment]
:+1:

**Comment by eldy** [comment]
When installing phpcs with composer, it seems it also install dolibarr dependencies libraries (exemple tcpdf 6.2.6). And of course, they are not version we want for the dolibarr we test. How can we be sure that the effect of line we see after instruction
composer require squizlabs/php_codesniffer:${PHPCS_VERSION}
does not affect dolibarr lib context ?

Is it possible to only load code sniffer ? May be by cleaning the composer.json file just before the composer require ?

**Comment by frederic34** [comment]
@eldy only phpcs is loaded now.

**Comment by rdoursenaud** [comment]
Composer should not pull the wrong versions of the dependencies.
The composer.json of each branch should list the correct version.
If the wrong version is pulled, there is a bug.

About the implementation:
I'd have used the composer.json "require-dev" rather than the "require" command to add php_codesniffer:

``` json
    "require-dev": {
        "squizlabs/php_codesniffer": "^2.0.0"
    },
```

And then `composer update squizlabs/php_codesniffer` rather than install to not touch anything else.

My 2 cents.

File: .travis.yml [hunk 1]
```diff
@@ -44,11 +45,18 @@ before_script:
...
+ - cp composer.json composer.json.old
+ - cp composer.json.phpcs composer.json
+ - composer self-update
+ - composer require squizlabs/php_codesniffer:${PHPCS_VERSION}
+ - composer install
  - phpenv rehash
- - which phpcs
- - phpcs --version
- - phpcs -i
+ - ln -s `pwd` htdocs/includes/squizlabs/php_codesniffer/CodeSniffer/Standards/PHPCompatibility
```
Comment (rdoursenaud) [inline .travis.yml:54]: What are you doing here?
This looks horribly wrong.

**Comment by frederic34** [comment]
I forgot to remove this strange test...

**Pull request merged**